src/components/MarkdownTextarea/index.tsx

```diff
@@ -62,23 +56,21 @@ export interface MarkdownTextareaProps {
      * included in a form submission. A disabled textarea cannot receive focus.
      * */
     disabled?: boolean;
+    /**
+     * Shows a preview that will be converted to html.
+     */
+    visible?: boolean
```

Comment:
Should this be preview?

Reply:
no. this has been removed to follow the 3rd API more closely using the visible prop